lib/output.js

```diff
@@ -694,6 +694,7 @@ function OutputStream(options) {
         output.semicolon();
     });
     DEFPRINT(AST_Toplevel, function(self, output){
+        use_asm = false;
```

Comment:
Anyway, triggering `use_strict` without tokens should be considered a bug for now. So I guess this (specific part) won't need fixing as it might relate to other places needing fixes instead.